models/DataObject/ClassDefinition/Dao.php

```diff
@@ -146,7 +146,7 @@ public function update()
               `id` BIGINT(20) NOT NULL PRIMARY KEY  AUTO_INCREMENT,
               `src_id` int(11) UNSIGNED NOT NULL DEFAULT '0',
               `dest_id` int(11) UNSIGNED NOT NULL DEFAULT '0',
-              `type` varchar(50) NOT NULL DEFAULT '',
+              `type` enum('object', 'asset','document') NOT NULL DEFAULT 'object',
```

Comment:
```suggestion
              `type` enum('object', 'asset','document') NULL DEFAULT NULL,
```
I'd let `NULL` be the default value, as `object` could lead to unexpected behavior. 
We'd then also need to change the migration script: 
- let `type` be `NULL`
- change all `''` (empy values)  to `NULL`
- update the column definition to enum

Reply:
@brusch like that?

Reply:
Exactly - thanks a  lot 👍